docs/source/acquire.rst

```diff
@@ -0,0 +1,31 @@
+acquire
+-------
+
+Acquire can **collect forensic artefacts** from a live system or an image. Resulting images
+can be fed to other Dissect tools like target-query.
+
+To run **acquire** on a live system:
+
+.. code-block:: console
+    
+    $ acquire
+
+.. note::
+
+    To be able to access the full system to collect all its artefacts one must
+    run acquire with administrator privileges.
+
+
+Profiles
```

Comment:
Why not specify which profiles there are and give a general description per profile?

Reply:
How can I specify which profile I want?

Reply:
Added some more details, but the full acquire documentation is already linked in the note.